### PR TITLE
fix(CgError) Show error message in log

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
When logging error messages with CgError as seen below, the actual error message is not shown.

#### Log

> Upload of case amusedhawk to GISAID unseccessful , case amusedhawk will be removed from delivery batch

### Fixed

- Fixes logging of error messages

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh fix-cg-errors`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by Vince-janv** [comment]
### With this branch:

> Upload of case amusedhawk to GISAID unseccessful No fasta file found for sample ACC10032A77, case amusedhawk will be removed from delivery batch

File: cg/exc.py [hunk 1]
```diff
@@ -127,8 +126,8 @@ class HousekeeperFileMissingError(CgError):
     Exception raised when a file is missing in Housekeeper.
     """
 
-    def __init__(self, message, errors=None):
-        self.message = message
+    def __init__(self, message: str = "", errors=None):
```
Comment (henrikstranneheim) [inline cg/exc.py:129]:
```suggestion
    def __init__(self, message: str = EMPTY_STRING, errors=None):
```
Matter of style - what do you think of using a named constant instead. It does read better.

We have some in cg/constants/symbols.py

**Review by henrikstranneheim — APPROVED** [review]
Good catch! 👍

File: cg/exc.py (hunk 1 above)
Comment (Vince-janv) [reply cg/exc.py:129]: I agree for those that are currently in `cg/constants/symbols.py` but for an empty string i quite like `""`. I feel like I would be more confused by `EMPTY_STRING`. I don't have strong opinion though, so feel free to overrule me 😉

Comment (henrikstranneheim) [reply cg/exc.py:129]: Ok, let's keep it as it is. 👍

**Pull request merged**